packages/openapi-ts/src/types/config.ts

```diff
@@ -125,9 +125,19 @@ export interface ClientConfig {
          */
         name?: string;
         /**
-         * Customise the method name of methods within the service
-         */
-        methodNameBuilder?: (service: string, operationId: string) => string;
+         * Customise the method name of methods within the service. By default, operationName is used.
+         */
+        methodNameBuilder?: (
+          service: string,
+          /**
+           * Method name used by default.
+           */
+          operationName: string,
+          /**
+           * The operationId from OpenAPI specification.
+           */
+          operationId: string | null,
+        ) => string;
```

Comment:
If possible, I would like to pass an object to the methodNameBuilder, but for backward compatibility, I have added an additional argument.

I think this is the ideal style:
```ts
methodNameBuilder?: (
  service: string,
  operation: {
    name: string,
    id: string | null,
  }
) => string;

or

methodNameBuilder?: (
  args: {
    service: string,
    operationName: string,
    operationId: string | null,
  }
) => string;
```

Reply:
Can you pass a single operation object here? We'll write migration notes, it's not hard to migrate